sanic/handlers.py

```diff
@@ -47,9 +61,14 @@ def response(self, request, exception):
         :param exception: Exception to handle
         :return: Response object
         """
-        handler = self.handlers.get(type(exception), self.default)
+        handler = self.lookup(exception)
         try:
-            response = handler(request=request, exception=exception)
+            if handler:
+                response = handler(request=request, exception=exception)
+            else:
+                response = None
```

Comment:
Why not just set response originally equal to None and then you wouldn't have to have this `else` clause